packages/calypso-e2e/src/lib/components/comments-component.ts

```diff
@@ -95,11 +84,14 @@ export class CommentsComponent extends BaseContainer {
 			throw new Error( `Failed to select a comment. Please check the comment number or selector.` );
```

Comment:
This was in place because otherwise the TypeScript transpiler would complain that `commentToLike` was not assigned before being used.

I have worked around the removal of this clause by changing Line 54 to:
```
		let commentToLike!: ElementHandle;
```